platforms-serverless/azure-functions-linux/run.sh

```diff
@@ -2,6 +2,9 @@
 
 set -eux
 
+echo "DATABASE_URL:"
+echo "$DATABASE_URL"
```

Comment:
Do we still need this?

Reply:
Oh no, that is leftover from debugging 👍 Will remove.